react/features/app-settings/components/styles.js

```diff
@@ -23,13 +23,14 @@ export default createStyleSheet({
      */
     backIcon: {
         alignSelf: 'center',
+        paddingRight: 12,
         ...Platform.select({
             android: {
                 fontSize: 24,
                 padding: 8
             },
             ios: {
-                fontSize: 30
+                fontSize: 36
```

Comment:
Please use the "arrow" icon on both platforms and a consistent styling. (size 26 looks good IMHO)